code/parachain/runtime/picasso/src/xcmp.rs

```diff
@@ -293,12 +293,12 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ChannelInfo = ParachainSystem;
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
 	type WeightInfo = cumulus_pallet_xcmp_queue::weights::SubstrateWeight<Self>;
-	type ControllerOrigin = EnsureRootOrHalfNativeTechnical;
-	type ExecuteOverweightOrigin = EnsureRootOrHalfNativeTechnical;
+	type ControllerOrigin = EnsureRootOrTwoThirdNativeTechnical;
+	type ExecuteOverweightOrigin = EnsureRootOrTwoThirdNativeTechnical;
```

Comment:
i do not think this change is good.

Reply:
Checking with Omar regarding Technical to make it less strict